From f38fe57b404073c3b08b88b04a5d9d976b0b8620 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Tue, 26 Mar 2024 11:45:29 +0100
Subject: [PATCH] arm64: dts: st: Add button-user-1 on board stm32mp257f-dk

Both button named USER1 and USER2 are available for Linux.

Change-Id: I12e266709779f7f3395b7791e5242be62ac41a79
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369352
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index 2e208042228c..ed35bb12c445 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -82,6 +82,13 @@ dmic1_endpoint: endpoint {
 	gpio-keys {
 		compatible = "gpio-keys";
 
+		button-user-1 {
+			label = "User-1";
+			linux,code = <BTN_1>;
+			gpios = <&gpioc 5 GPIO_ACTIVE_HIGH>;
+			status = "okay";
+		};
+
 		button-user-2 {
 			label = "User-2";
 			linux,code = <BTN_2>;
-- 
2.39.5

